frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -8,13 +10,29 @@ export type FaqCategory =
   | "After Sending Your Letter"
   | "States with Limited Protections";
 
+type FaqCategoryLabels = {
+  [k in FaqCategory]: string;
+};
+
+export function getLabelNameForFaqCategory(): FaqCategoryLabels {
+  return {
+    "Letter Builder": li18n._(t`Letter Builder`),
+    "Tenant Rights": li18n._(t`Tenant Rights`),
+    "Connecting With Others": li18n._(t`Connecting With Others`),
+    "After Sending Your Letter": li18n._(t`After Sending Your Letter`),
+    "States with Limited Protections": li18n._(
+      t`States with Limited Protections`
+    ),
```

Comment:
wait I just realized these are in title case too lol.  I thought we were using sentence case!!

Reply:
Hmmm, since these are labels they should actually be in title case (see https://github.com/JustFixNYC/tenants2/pull/1405#issuecomment-627415847 below), but also these one's are special and actually get transformed into all CAPS. So, if it feels better to convert them to sentence case, I can do so.